src/pages/content.tsx

```diff
@@ -282,7 +283,7 @@ const Content = () => {
                                 onCancelInput={console.log}
                             />
                         )}
-                        <SearchBar />
+                        {connectConfig.content.showSearchBar && <SearchBar />}
```

Comment:
Can we move this into a useConnectConfig hook? So it is more react-friendly and supports dynamic values

Reply:
sure! 
Added in this commit: [0394698](https://github.com/powerhouse-inc/document-model-electron/pull/185/commits/0394698fb236f061c9baff7365db3f4ed8674712)